src/main/java/com/enterprise/redcord/RedCordController.java

```diff
@@ -47,14 +47,14 @@ public String index(@ModelAttribute("topicEntry") String topicName, Model model)
             topicId = topicService.fetchByTopicName(topicName);
             messages = messageService.fetchByTopicId(topicId);
             topics = topicService.fetchAllTopics();
+            model.addAttribute("messages", messages);
+            model.addAttribute("topics", topics);
         } catch (ExecutionException e) {
             e.printStackTrace();
         } catch (InterruptedException e) {
             e.printStackTrace();
         }
```

Comment:
While you're in here, this is something that can be cleaned up in code review.  You can implement best practices in exception handling.

Use proper logging instead of printStackTrace().
- Logging has the concept of priority (error, debug, warn, etc.), which printStackTrace doesn't have.
- No one will see printStackTrace(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with printStackTrace.